source/components/namespace/nsalloc.c

```diff
@@ -237,6 +237,10 @@ AcpiNsDeleteNode (
         return_VOID;
     }
 
+#ifdef ACPI_USE_ACCELERATED_NS_SEARCH
```

Comment:
This should probably be `ACPI_USE_NS_SEARCH_ACCELERATION` :)

Reply:
Good catch.